app/views/track/_tracking_links_simple.html.erb

```diff
@@ -0,0 +1,15 @@
+<%
+  existing_track = local_assigns.fetch(:existing_track) do
+    TrackThing.find_existing(@user, track_thing) if @user
+  end
+%>
+
+<% if existing_track %>
```

Comment:
Is that because it only hides the button and not the container too? If it hid ```  <div class="feed_link feed_link_<%= location %>">``` as well it would collapse in, it would be a bit tricky to make it collapse in any other way